api/src/test/java/org/openmrs/api/VisitServiceTest.java

```diff
@@ -955,6 +980,34 @@ public void saveVisit_shouldNotPersistNewEncounter() {
 		Context.flushSession();
 	}
 
+	/**
+	 * @see VisitService#purgeVisit(Visit) 
```

Comment:
please remove all javadocs you added for the tests since they
* do not provide additional information. We can see/navigate to the method under test by looking inside the test. Its better to make the test readable by using spaces to separate the test setup/arrangement, the action which is the method under test, and the assertions which are things you expect to happen after the action. Think of it as AAA - 3 blocks of arrange, act, assert.
* just take up space
* will be out of date if the method under test changes signature and no one will update the huge number of javadocs. We have been/or better are in this situation if you look at our code :) This leads to a lot of errors shown by the IDE which just clutter and take up our attention.